pop-miners/altchain-pop-miner/src/main/kotlin/org/veriblock/lite/NodeCoreLiteKit.kt

```diff
@@ -98,26 +79,40 @@ class NodeCoreLiteKit(
         }
     }
 
-    fun getAddress(): String = addressManager.defaultAddress.hash
+    fun balanceUpdater() {
+        GlobalScope.launch {
+            while (isActive) {
+                try {
+                    updateBalance()
+                } catch (e: Exception) {
+                    logger.error { e.message; e }
```

Comment:
```suggestion
                    logger.error(e) { e.message }
```
You can pass the 'e' like that, also the message should be a little more descriptive/graceful than the exception message)